### PR TITLE
Feature/buttons

### PR DESCRIPTION
**Pull request by omarawad0**
i have added the buttons section, form and tabs

### CODE REVIEW

**Comment by Nilegfx** [comment]
this is not a review but just an overall comment.

Your commits are too granular (too mnay commits), try to combine them into set of workable commit.

**Comment by omarawad0** [comment]
Okay i will try.

File: tailwind.config.js [hunk 1]
```diff
@@ -109,6 +108,26 @@ module.exports = {
...
+        xl: '2rem',
+        xxl: '2.5rem',
+        xxl1: '2.875rem',
+        '3xl': '3rem',
+        '3xl1': '3.625',
+        '4xl': '4rem',
+        '5xl': '8rem'
+      },
+      backgroundOpacity: {
+        24: '0.24',
```
Comment (saifaldin) [inline tailwind.config.js]: Please revise the need for all of these values.
For example, the lengths of input fields don't have to be exactly as the design of atom, the lengths might change when put in context, molecules and organisms.
Also, some of these already exist in tailwind, like `8rem` and `2rem`. We are only extending here so they are still there.

**Review by saifaldin — CHANGES_REQUESTED** [review]
Great attention to detail. But please consider the following:
- Adding focus state styles for the buttons.
- Making the entire radio button clickable, not just the little circles.
- Improving the dimensions of the radio button circle fill in.
- Last, you have deleted most, if not all, of the classes added by Kareem to the file `buttonswithTW.css`, probably while resolving a conflict. Please find them and paste them back in.

**Review by saifaldin — APPROVED** [review]
LGTM

**Pull request merged**